### ISSUE TITLE
mismatch function argument between cassandra_cql and cassandra_cql_return

### ISSUE THREAD

**Issue by GwiYeong** [saltstack/salt#56328]
### Description of Issue
there is mismatch keyword argument between `cassandra_cql` and `cassandra_cql_return`

the function [cql_query_with_prepare](https://github.com/saltstack/salt/blob/master/salt/modules/cassandra_cql.py#L365) is defined like below

```python
def cql_query_with_prepare(query, statement_name, statement_arguments, callback_errors=None, contact_points=None, port=None, cql_user=None, cql_pass=None, **kwargs):

    # Backward-compatibility with Python 3.7: "async" is a reserved word
    asynchronous = kwargs.get('async', False)
```

the first line of the function gets `async` param from `**kwargs`. 

however the returner, `cassandra_cql_return` calls this function with argument that is named `asynchronous`.

See the below code line.

https://github.com/saltstack/salt/blob/master/salt/returners/cassandra_cql_return.py#L205

https://github.com/saltstack/salt/blob/master/salt/returners/cassandra_cql_return.py#L226

https://github.com/saltstack/salt/blob/master/salt/returners/cassandra_cql_return.py#L268

https://github.com/saltstack/salt/blob/master/salt/returners/cassandra_cql_return.py#L298

these functions call `cassandra_cql.cql_query_with_prepare` with `asynchronous=True`, but actually `cql_query_with_prepare` not work asynchronous

maybe missing in this commit https://github.com/saltstack/salt/commit/377fbbc7b760bad1fd267747bf463e7cc8f3c1ae

### Setup
None

### Steps to Reproduce Issue
just see code.

### Versions Report
master branch

**Comment by Ch3LL** [comment]
nice catch. seems you have a handle here, would you mind sending in a PR with a test case as well?

**Comment by GwiYeong** [comment]
@Ch3LL 
thanks to response. Ok, I will.
should I send pr to the `master` branch?

**Comment by GwiYeong** [comment]
@Ch3LL hi I found this commit 377fbbc7b760bad1fd267747bf463e7cc8f3c1ae. I could not understand this commit. 
since python 3.7, `async` is reserved keyword, so use `asynchronous` for function args is correct. 
mayby this issues will fixed after rollback 377fbbc7b760bad1fd267747bf463e7cc8f3c1ae commit
can you give me a comment?

**Comment by Ch3LL** [comment]
yep master. looks like you got that right :) i'll leave comments on your PR. thanks for this!
